include/deal.II/fe/fe_tools.h

```diff
@@ -1135,7 +1135,7 @@ namespace FETools
    * function, use the add_fe_name() function.  This function does not work if
    * one wants to get a codimension 1 finite element.
    */
-  template <int dim, int spacedim>
+  template <int dim, int spacedim=dim>
```

Comment:
This is not legal C++03. Since the function is new, we need not worry about backward compatibility. So how about doing something like this:
```
  template <int dim, int spacedim
#ifdef DEAL_II_WITH_CXX11   // or whatever the appropriate tag is
    = dim
#endif
      >
```
? This works for all with modern compilers. For those who are backward, they need to write a few extra characters. And we get to remove the workaround again in a couple of years at the latest.

You'd have to undo the changes in the other places, though, where you call the function with only one argument because it needs to be able to compile with C++98 for the moment.